generator-scrivito/generators/widget/index.js

```diff
@@ -0,0 +1,61 @@
+const Generator = require("yeoman-generator");
+const capitalizeFirstLetter = require("../../utils/capitalizeFirstLetter");
+
+module.exports = class extends Generator {
+  start() {
+    this.prompt({
+      type: "input",
+      name: "nameWidget",
+      message: "Enter a name of the Widget: ",
+    }).then(answers => {
+      const defaultName = answers.nameWidget;
+      const defNameUpper = capitalizeFirstLetter(defaultName);
+      const defNameUpperWidget = `${defNameUpper}Widget`;
```

Comment:
How about making a small helper that converts an input such as `MyHeadlineWidget` to `My Headline`?

```js
      const humanFriendlyName = lodash.startCase(
        widgetClassName.replace(/Widget$/, "")
      );
```

This of cause requires a dependency to `lodash`:

```
const lodash = require("lodash");
```

And that needs to be part of `generator-scrivito`. So in `scrivito_example_app_js/generator-scrivito` call `npm i lodash`, which adds `lodash` as an official dependency of `generator-scrivito`.

`humanFriendlyName` is only used in `XWidgetEditingConfig.js.ejs`:
```ejs
title: "<%= humanFriendlyName %>",
```